### ISSUE TITLE
Remove cyclical dependency

### ISSUE THREAD

**Issue by Matt-Butler** [cerner/terra-core#417]
# Cyclical dependency in monorepo

```
lerna WARN ECYCLE Encountered a cycle in the dependency graph. This may cause instability!
```
terra-base has a dependency on terra-i18n and terra-i18n has a dependency on terra-base.
1. https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/package.json#L36
2. https://github.com/cerner/terra-core/blob/master/packages/terra-base/package.json#L32

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [ ] Bug
- [x] Other

## Steps to Reproduce

1. Run `npm install`

**Comment by Matt-Butler** [comment]
Since terra-base is a hoc of terra-i18n, I think we should remove there terra-base dependency in terra-i18n.

**Comment by Matt-Butler** [comment]
Resolved in #418
